isis/src/socet/apps/socetframesettings/socetframesettings.cpp

```diff
@@ -422,24 +430,228 @@ void IsisMain() {
+// this application.
+
+
+// NOTES
+
+// 0) The definition of the SOCET SET (SS) plate reference frame is virtually
+//    undocumented in the public domain.  From what is available, it appears
+//    that
+//
+//    i) +zSS is the anti-boresight (normal to the SS focal plane away from the
+//       direction of the imaged scene)
+//
+//    ii) +xSS is typically displayed to the right, and +ySS is displayed up
+//        in the right-handed system, HOWEVER ...
+//
+//    iii) It may be that the orientations of +xSS and +ySS are dependent on
+//         the SOCET cam file used; that file is external to this appication,
+//         and the per-spacecraft and/or per-instrument if-else clauses in
+//         this application make assumptions about the contents of that
+//         external file.
+
+// (1) Planetographic coordinates are referred to as geodetic coordinates
+//     in SPICE.  Planetographic latitude and height are normal and
+//     relative, repsectively, to the surface of a target modeled as a
+//     spheroid that is a volume of revolution.  Only the radius at the
+//     intersection of the prime meridian and the equator, and the polar
+//     radius, are used; the intermediate radius, at the intersection of
+//     meridians +/-90degrees and the equator, of the tri-axial ellipsoid
+//     model is not used.
+//
+//     N.B. Planetographic coordinates are used ***ONLY*** for calculating
+//          the camera position; they are ***NOT*** used in not used in
+//          the calculation of the camera attitude OPK angles.
+
+// (2) Specifically, LsrToSsMatrix = [kappa]  [phi]  [omega]
+//                                          3      2        1
+//     where
+//
+//       [angleN]
+//               axisN
+//
+//     represents a *FRAME* rotation by angleN about the coordinate axis
+//     indexed by axisN.  N.B. the result of one
+//
+//       [angleN]
+//               axisN
+//
+//     frame rotation is a matrix, which rotates vectors by -angleN radians
+//     about the axisN coordinate axis cf. SPICE RECGEO documentation 
+//
+//       https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/cspice/recgeo_c.html
+
+// (3)  LSR frame - Local Space Rectangular frame
+//      - A.k.a. ENU frame, East-North-Up (What's up?  East cross North)
+//      The LSR frame is based on the position of the instrument in the
+//      target body-fixed (BF) frame, where
+//
+//      +zLSR => Vector from target body cent to camera position. N.B. this
+//               vector is planetocentric and different from the
+//               plantographic (geodetic) coordinates described in (1) above.
+//
+//      +xLSR => Direction of east longitude at sub-camera point, equal to
+//               cross product of planetocentric North polar axis (+zBF)
+//               vector with +zLSR vector (above).
+//
+//      +yLSR => In same half-plane of xzLSR plane as +zBF (north, or
+//               positive rotation, pole)
```

Comment:
I'd like to see this whole section of comments converted to doxygen style function documentation. We don't distribute doxygen documentation for functions in application code, so I would be okay with just converting to a block comment instead of single line comments if going completely to doxygen would be a significant amount of work.

Reply:
go for it!